File: src/ItemCommander.EntityService/Models/ItemCommanderResponse.cs
using System.Collections.Generic;

namespace ItemCommander.EntityService.Models
{
    public class ItemCommanderResponse
    {
        public string CurrentPath { get; set; }

        public string CurrentId { get; set; }

        public string ParentId { get; set; }

        public List<ItemResponse> Children { get; set; }
    }

    public class DownloadResponse
    {
        public string FileName { get; set; }
    }
}

File: src/ItemCommander.Client/src/app/contract/ItemCommanderResponse.ts
import { Item } from "./Item";

export interface ItemCommanderResponse {
    CurrentPath: string;
    ParentId:string;
    Children: Array<Item>; 
    CurrentId: string;
}



export interface FieldDto
{
    Id:String;
    Name:string;
    Value:string;
    Type:string;
    SectionName:string;
}
